react-common/styles/controls/Modal.less

```diff
@@ -110,6 +118,10 @@
         }
     }
 
+    .common-modal-back:focus-within {
+        border: 1px, solid var(--pxt-primary-foreground);
```

Comment:
Would --pxt-focus-border be more appropriate here?

Reply:
Sure, I can see how this looks!